### PR TITLE
Student home

### LINKED ISSUE #79: [FEATURE] - Student Home
### Summary

The home page for students where they should be able to navigate to:
- Classroom the student is a part of
- Session the student is participating in
- Join new classroom/session view
 
### Definition of Done

When a student is redirected to studenthome, and can access the classroom, session and join new classroom from here.

### PR DESCRIPTION
**Pull request by andreaswurtz**
## Description

Routes student to /studenthome, that contains timedSession and classroom info for a student.

### Resolved Issue
Fixes #79

### CODE REVIEW

File: src/lib/fetchRequests.ts [hunk 1]
```diff
@@ -48,6 +48,20 @@ export async function fetchSpecificSession(
 	});
 }
 
+export async function fetchClassroomSession(
```
Comment (EMojoKrogh) [inline src/lib/fetchRequests.ts:51]: ah ah ah you didn't say the magic word.
duplicate kode:
image

Comment (EMojoKrogh) [reply src/lib/fetchRequests.ts:51]: men navnet på din er måske bedre

File: src/routes/api/update-session-status/+server.ts [hunk 2]
```diff
@@ -21,17 +22,22 @@ export const POST: RequestHandler = async ({ request, cookies }) => {
 		cookies
 	);
 
+	debugUpdateSessionStatus(response);
+
 	// Ensure that a response is returned in all cases
 	if (response.ok) {
 		const session = await response.json();
```
Comment (EMojoKrogh) [inline src/routes/api/update-session-status/+server.ts:29]: fjern den console.log lad mig høre dig sige

**Review by EMojoKrogh — CHANGES_REQUESTED** [review]
some changes needed

File: src/routes/api/update-session-status/+server.ts (hunk 2 above)
Comment (Schoogle) [reply src/routes/api/update-session-status/+server.ts:29]: indeed

Comment (EMojoKrogh) [reply src/routes/api/update-session-status/+server.ts:29]: Never mind

**Review by EMojoKrogh — APPROVED** [review]
Looks good

**Pull request merged**